app/backend/src/couchers/models.py

```diff
@@ -15,7 +15,7 @@
     naming_convention={
         "ix": "ix_%(column_0_label)s",
         "uq": "uq_%(table_name)s_%(column_0_name)s",
-        "ck": "ck_%(table_name)s_%(constraint_name)s",
+        "ck": "ck_%(table_name)s",
```

Comment:
Why change this? I think it'll mess up the migrations

Reply:
We don't give names to all our constraints, I got complaints when running with sqlite as backend, but I guess it is DB independent. I can revert it since I'm not fully up to date with how the migrations work.